pkg/k8s/apis/cilium.io/v2/client/register.go

```diff
@@ -95,13 +95,18 @@ func CreateCustomResourceDefinitions(clientset apiextensionsclient.Interface) er
 		return createIdentityCRD(clientset)
 	})
 
+	// Reviewers: I'm not sure if it makes sense to wrap this under the feature
+	// flag check since we have related Makefile changes.
+	// if option.Config.EnableLocalRedirectPolicy {
 	g.Go(func() error {
 		return createCLRPCRD(clientset)
 	})
+	// }
 
-	g.Go(func() error {
-		return createCCLRPCRD(clientset)
-	})
+	// CCLRP support will be added in future release.
+	// g.Go(func() error {
+	// 	return createCCLRPCRD(clientset)
+	// })
```

Comment:
From what I have gathered from others on this PR is that CCLRP were never implemented, but we added functions like this anyway? I presume that also explains the Makefile change as well. 

I agree with Joe. Let's' remove this CCLRP code and the Makefile change in a dedicated commit. Please also mention in that commit that it will eventually be reverted once the functionality of the CCLRP is implemented.

Reply:
> Please also mention in that commit that it will eventually be reverted once the functionality of the CCLRP is implemented.

Although this would be good detail to include, I'm not sure that this is the case. I had a chat with @aditighag about this a while back and I don't think we were able to point out a specific use case or reason for clusterwide LRP.

Reply:
> > Please also mention in that commit that it will eventually be reverted once the functionality of the CCLRP is implemented.
> 
> Although this would be good detail to include, I'm not sure that this is the case. I had a chat with @aditighag about this a while back and I don't think we were able to point out a specific use case or reason for clusterwide LRP.

Ah OK, I wasn't aware of that. Cool.